lib/views/help/requesting.es.html.erb

```diff
@@ -383,10 +383,7 @@
         </li>
       </ol>
       <p>
-        En estos casos puedes utilizar los recursos legales planteados en el punto 19.
-      </p>
-      <p>
-        Tendrás la posibilidad de categorizar tu respuesta según una lista de categorías (ver punto 17). Si no sabes cómo hacerlo te ayudaremos. <i>derechoapreguntar.org</i>
+        En estos casos puedes utilizar los recursos legales planteados en esta web: <a href="#recursos">Los Recursos en Nicaragua</a>
```

Comment:
Ah! ok, I think I may have named something else recursos that wasn't named it before, does that make sense? So they wanted that to link to a section further down, and I made the anchor tag myself and named them both recursos